src/main.cpp

```diff
@@ -46,7 +49,8 @@ namespace
             ("input,i", po::value<std::string>()->default_value("stdin"), "Path to the input VCF file, or stdin")
             ("level,l", po::value<std::string>()->default_value("warning"), "Validation level (error, warning, stop)")
             ("version,v", po::value<std::string>(), "VCF fileformat version to validate the file against (v4.1, v4.2, v4.3)")
-            ("output,o", po::value<std::string>()->default_value("stdout"), "Comma separated values for type of output (database, stdout, silent)")
+            ("report,r", po::value<std::string>()->default_value("stdout"), "Comma separated values for types of reports (database, stdout)")
+            ("outdir,o", po::value<std::string>()->default_value(""), "Directory for the output (same as input file)")
```

Comment:
Do I add an explanation in the message? or leave it as just "Directory for the output" and develop the explanation in the readme?